src/strain.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * strain.c
+ *
+ *  Created on: 31 Mar 2021
+ *      Author: nikbaya
+ */
+
+#include "strain.h"
+#include "model.h"
+#include "utilities.h"
+
+/*****************************************************************************************
+*  Name:		initialise_strain
+*  Description: Initialises new strain, can only be called once for each strain
+*  Returns:		void
+******************************************************************************************/
+void initialise_strain(
+	model *model,
+	long idx,
```

Comment:
I don't think this should take index. It should always increment by adding to end of the list and return the index of the newly initialised strain.